libraries/Parsers/Microsoft.Bot.Builder.Parsers.LU/Parser/EntitySection.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+using System.Collections.Generic;
+
+namespace Microsoft.Bot.Builder.Parsers.LU.Parser
+{
+    /// <summary>SectionEntity class.</summary>
+    public partial class SectionEntity
```

Comment:
Does tihs need to be public? Can we make it internal? Ideally we'd make as many classes internal as we possibly cna, and leave the publi surface as small as possible, just to keep in mind.

Reply:
Will consider making all the models internal, we can discuss it with @munozemilio